src/utils/context_utils.py

```diff
@@ -71,7 +71,7 @@ def get_story_text_generator(
         game_state = get_game_state(context)
         preferences = game_state.preferences
 
-        open_ai_models = ["gpt-3.5-turbo", "gpt-4"]
+        open_ai_models = ["gpt-3.5-turbo", "gpt-4-1106-preview"]
```

Comment:
question: do we no longer wish to support the non-turbo gpt-4?

i was expecting:

```python
open_ai_models = ["gpt-3.5-turbo", "gpt-4", "gpt-4-1106-preview"]
```

Reply:
Two thoughts:

- From what I understand, the gpt4-turbo will replace the use of gpt-4, since it's supposed to be all of better/faster/cheaper.
- We ought ot just remove this layer of 'approved' models, since there's no user-facing way to change it, nor do we intend there to be.

Reply:
i'll be curious to see if it actually does fully replace `gpt-4`, or if we'll find out that `gpt-4` classic is actually still better in some significant fashion (like, oh, you want to do X?  that was nerf'd in turbo)